### DISCUSSION TITLE
not supporting subprocess mudol

### DISCUSSION THREAD

**Discussion by NasserMashpery** [TomSchimansky/CustomTkinter#2828] [Q&A]
i want to imnport another python file but not working with my project please help

**Comment by dipeshSam** [comment]
Could you please provide Sample Reproducible Code?

Regards.

**Comment by NasserMashpery** [comment]
![Screenshot 2024-07-19 133946](https://github.com/user-attachments/assets/c300e1ea-7a35-4cb3-a088-d70b2c2648f4)
![Screenshot 2024-07-19 134002](https://github.com/user-attachments/assets/2e1d0922-5531-449a-8178-650cc839f647)

**Comment by NasserMashpery** [comment]
![image](https://github.com/user-attachments/assets/e07c4c09-6a64-4f93-a8f5-1a1ed8a03a64)
![image](https://github.com/user-attachments/assets/672f2171-f26a-41b0-9fe8-9b3910ec3a94)

**Comment by NasserMashpery** [comment]
i want import the test to test2 but not working but whene i run test with (TK) in test (CTk) its working

**Comment by NasserMashpery** [comment]
why not someone answer me here ?

**Comment by dipeshSam** [comment]
Have you tried to use full path?? It might be helpful.

Regards.

**Comment by NasserMashpery** [comment]
I will check it mr samarwat and inform you

Regards ;


احصل على Outlook لـ iOS<https://aka.ms/o0ukef>
________________________________
من: Dipesh Samrawat ***@***.***>
‏‏تم الإرسال: Saturday, July 20, 2024 6:46:10 AM
إلى: TomSchimansky/CustomTkinter ***@***.***>
نسخة: NasserMashpery ***@***.***>; Author ***@***.***>
‏‏الموضوع: Re: [TomSchimansky/CustomTkinter] not supporting subprocess mudol (Issue #2508)


Have you tried to use full path?? It might be helpful.

Regards.
